src/ptvsd/_vendored/pydevd/pydevd_plugins/django_debug.py

```diff
@@ -346,8 +356,21 @@ def _is_django_variable_does_not_exist_exception_break_context(frame):
 #=======================================================================================================================
 
 
-def can_not_skip(plugin, main_debugger, frame):
-    return main_debugger.django_breakpoints and _is_django_render_call(frame)
+def can_skip(plugin, main_debugger, frame):
+    if main_debugger.django_breakpoints:
+        if _is_django_render_call(frame):
+            return False
+
+    if main_debugger.django_exception_break:
+        module_name = frame.f_globals.get('__qualname__', '')
```

Comment:
I don't think modules can have `__qualname__`. There was a PEP for that, but it was withdrawn.